flasgger/base.py

```diff
@@ -215,6 +215,8 @@ def get(self):
             data['host'] = self.config.get('host')
         if self.config.get("basePath"):
             data["basePath"] = self.config.get('basePath')
+        if self.config.get("securityDefinitions"):
+            data["securityDefinitions"] = self.config.get('securityDefinitions')
```

Comment:
This line is failing on tests because line is longer than 79 chars. can you change to

```python
data["securityDefinitions"] = self.config.get(
    'securityDefinitions'
)
```

we still use 79 chars limit because some of us use VIM and Emacs.